framework/CodeInterfaces/RELAP5/relapdata.py

```diff
@@ -157,6 +153,20 @@ def readMinorBlock(self,lines,i):
           flagg2=1
     return minorDict
 
+  def checkLine(self,list):
+    """
+      Method that check the content of a list (i.e., a line); a list must contain only numbers
+      @ In, list, list, list that contained values located in a single line
```

Comment:
```list``` is a built-in datatype in Python. Even if you can assign a variable name to that, it is generally discouraged. Can you change the variable name to something else? (e.g. ```lineList```)